packages/shared/src/components/drawers/Drawer.tsx

```diff
@@ -116,11 +121,16 @@ function BaseDrawer({
           {children}
         </ConditionalWrapper>
         {displayCloseButton && (
-          <div className="sticky -bottom-3 bg-background-default">
+          <div
+            className={classNames(
+              'sticky -bottom-3 bg-background-default',
+              className?.close,
+            )}
+          >
             <Button
               variant={ButtonVariant.Float}
               className="mt-3 w-full"
-              onClick={onClose}
+              onClick={(e) => onClose(e.nativeEvent)}
```

Comment:
Why was this change needed?

Reply:
Previously, the type was not needing to pass an event.